src/DataGrid.tsx

```diff
@@ -769,13 +909,13 @@ function DataGrid<R, K extends keyof R, SR>({
 
   return (
     <div
-      role="grid"
+      role={hasGroups ? 'treegrid' : 'grid'}
```

Comment:
Perhaps we need to expose this `role` props outside so that people that are using other languages can set the value as well in different language.

Reply:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques

I don't think aria-roles need to be translated.

Reply:
You are right. I was thinking about `aria-label` for screen reader.